src/containers/Header/Header.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import styles from './Header.styles';
+
+export default class Header extends React.Component {
+    render() {
+        return <div style={styles.header}>Harrison</div>
```

Comment:
Why is it just Harrison here? Presumably it's a placeholder? If so, let's add a comment here with a TODO